### ISSUE TITLE
1.0.8 results in whitescreen due to missing class MsRobotstxtManager_Admin

### ISSUE THREAD

**Issue by mundschenk-at** [tribalNerd/multisite-robotstxt-manager#3]
After installing version 1.0.8, the backend fails:

`PHP Fatal error:  Uncaught Error: Class 'MsRobotstxtManager_Admin' not found in /<path>/wp-content/plugins/multisite-robotstxt-manager/multisite-robotstxt-manager.php:101`

**Comment by tribalNerd** [comment]
Hello mundschenk-at,

That error is because the admin url (file name) to the plugin changed. After plugins upgrade, the WP Admin Area doesn't refresh, so the old admin area link still exists in the settings menu,  until you manually refresh.

I'll try to figure out a solution to that, maybe I can redirect the old url to the new.

With that, I'll get the previous versions published on git as well, thanks for tip.

Chris

**Comment by mundschenk-at** [comment]
I don't think that's it. The problem was not a link to a non-existing PHP file. Rather, the whole backend fails (whitescreen). There could be a relation PHP-FPM opcode caching (though my system is set to reread the file if the mtime has changed).

**Comment by tribalNerd** [comment]
That particular class is the old admin area class, the new class is: MsRobotstxtManager_AdminArea, and line 101 within the new version of multisite-robotstxt-manager.php is a blank line. I tested a fresh download/install on my local dev, with no issues. When I tested it on the live dev, I did an upgrade, when I accessed the plugin admin I got the error "page does not exist."

I don't think I can do a refresh of PHP-FPM via the plugin, however I can look into that.

Chris
